shard.override.yml

```diff
@@ -3,9 +3,9 @@
 
 # NOTE: https://github.com/crystal-lang/shards/issues/432
 # Until this is resolved, the default needs to be empty here.
-dependencies: {}
+#dependencies: {}
 # Uncomment if you need to override
-# dependencies:
-#   lucky:
-#     github: luckyframework/lucky
-#     branch: master
+dependencies:
+  lucky:
+    github: luckyframework/lucky
+    branch: master
```

Comment:
I think you'll need to add avram to this list since lucky isn't pointing at the master branch

Reply:
Yeah, that wasn't working. Even though I had avram overridden, it seemed that it was still being superseded by the latest release. 🤷